src/pages/kb/user-guide/alerts/setting-up-an-alert.md

```diff
@@ -15,54 +15,76 @@ title: Setting Up An Alert
 slug: setting-up-an-alert
 order: 1
 ---
-Redash alerts can notify you when your [**Scheduled Queries**]({% link _kb/user-guide/querying/scheduling-a-query.md %}) match some arbitrary criteria. Use them to monitor business data or integrated them with tools like Zapier or IFTTT to trigger business processes like user onbarding or support tickets. While Alerts are usually combined with scheduled queries, the Alert criteria will be evaluated every time the query is executed.
+
+
+Redash alerts notify you when a field returned by a [**Scheduled Query**]({% link _kb/user-guide/querying/scheduling-a-query.md %}) meets a threshhold. Use them to monitor your business. Or integrate them with tools like Zapier or IFTTT to kickoff workflows such as user onbarding or support tickets. Alerts complement scheduled queries, but their criteria are checked after every execution.
+
+{% callout warning %}
+
+For information on alerts prior to Redash V9, see our [**Legacy Alerts doc**](setting-up-an-alert-v8).
+
+{% endcallout %}
 
 {% callout warning %}
 
-A query schedule is not mandatory but is _highly recommended_ for alerts. If you configure an Alert without a query schedule you will only receive notifications if a user in your organization manually executes the given query.
+A query schedule is not required but is _highly recommended_ for alerts. If you add an alert to a non-scheduled query you will be notified only if a user executes the query manually and the alert criteria are met.
 
 {% endcallout %}
 
-Click `Alerts` on the main navigation menu to see a list of the existing Alerts.
+To see a list of current Alerts click **Alerts** on the navbar. By default, they are sorted in reverse chronological order by the **Created At** column. You can reorder the list by clicking the column headings.
+
+* **Name** shows the string name of each alert. You can change this at any time.
+* **Created By** shows the user that created this Alert.
+* **State** shows whether the Alert status is `UNKNOWN`, `TRIGGERED`, or `OK`.
 
 ![](/assets/images/docs/gitbook/alerts.png)
 
 # Usage
 
-Create a new Alert by clicking the blue `Create` button in the navigation menu and selecting `Alert`.
+Click the **Create** button in the navbar and then click **Alert**.
 
 ![](/assets/images/docs/gitbook/create-alert.png)
```

Comment:
@susodapop this image also needs an update

Reply:
This is merged, but can be updated in another PR.